tests/app/views/test_suppliers.py

```diff
@@ -1350,6 +1316,36 @@ def test_get_supplier_framework_info(self):
             'uploaderUserId': 20
         }
 
+
+class TestSupplierFrameworkUpdates(BaseApplicationTest, JSONUpdateTestMixin):
+    method = "post"
+    endpoint = "/suppliers/0/frameworks/digital-outcomes-and-specialists"
```

Comment:
Sorry, could you please explain what this is for? Haven't been able to quite work it out. I see it somehow relates to JSONUpdateTestMixin but it's interesting that this is for DOS but not all of this class's tests are about DOS (at least one is doing things with G-cloud 8).